docs/usermanual.md

```diff
@@ -116,10 +116,47 @@ tools available online like [this one](https://string-functions.com/string-hex.a
 
 It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with
 `payloadType` `ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure
-attributes and `id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used,
-as well as `actionType`.
+attributes.
 
-Examples of these `ngsiv2` payloads are the following ones:
+Note that the entity ID and type in the measure are also include as attributes `measure_id` and `measure_type` as described 
+[here](https://github.dev/telefonicaid/iotagent-node-lib/doc/api.md#special-measures-and-attributes-names) (both using 
+attribute type `Text`). The ID and type of the entity updated at Context Broker is taken from device/group configuration or provision,
+
+However, it is possible to use the same entity ID that the original one by using `entityNameExp` 
+at [device group provision](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#config-group-datamodel), this way:
+
+```
+"entityNameExp": "measure_id"
+```
+
+The `actionType` used in the update sent to Context Broker is taken from the measure in the case that measure corresponds to
+a NGSI-v2 batch update. In other cases (i.e. NGSI-LD or NGSI-v2 non-bath update), the `actionType` is the default one (`append`).
```

Comment:
I have tried to clarify how actionType is established, but I have some doubts.

- The possibility of taking the actionType from the measure is only available in NGSI-v2 batch operation. In other words, it is not available in NGSI-LD (no matter if batch or not) and NGSI-v2 update operations different from batch.
- The default actionType is not configurable at the present moment (I think in the past it was, using the `appendMode` or similar in the configuration) and it's alwasys `append`.  My understanding is based in [this code](https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/ngsi/entities-NGSI-v2.js#L539) but I don't know if I have interpreted it correctly...

Reply:
Looks good